react/Pill/Pill.less

```diff
@@ -0,0 +1,87 @@
+@import (reference) '~seek-style-guide/theme';
+
+@staticTagHeight: @row-height * 5;
+@interactiveTagHeight: @row-height * 8;
+@removeCircleDiameter: @row-height * 4;
+
+.staticPill {
+  .rawText(@interaction-type-scale);
+
+  align-items: center;
+  display: inline-flex;
+  color: @sk-charcoal;
+  background: @sk-gray-light;
+  border-radius: @staticTagHeight / 2;
+  height: @staticTagHeight;
+  line-height: @staticTagHeight;
+  padding: 0 15px;
+  margin: @row-height 10px @row-height 0;
+}
+
+.interactivePill {
+  align-items: center;
+  display: inline-flex;
+  font-size: 18px;
+  background: @sk-gray-light;
+  border-radius: 24px;
```

Comment:
Should this be more dynamic? Maybe the same value as `@removeCircleDiameter`? or is that a coincidence?

Reply:
Good catch, the radius should be half the height of the interactive pill so I've changed it to `@interactiveTagHeight / 2`